lib/utils/caching.ts

```diff
@@ -0,0 +1,2 @@
+export const APP_CACHE_KEY = "app-cache-violet";
+export const WORKSPACE_ID_COOKIE_NAME = `${APP_CACHE_KEY}-workspace-id`;
```

Comment:
I made this change to invalidate the current cookie, and also tie it to the app cache whenever we need to rotate that.

Reply:
If you think it's overkill to link these two @brandonroberts @jpmcb, I can just keep them separate.

```suggestion
export const WORKSPACE_ID_COOKIE_NAME = "workspace-id";
```

Reply:
I'm good with how it its, I don't have a strong opinion about it